flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -284,5 +233,63 @@ var A320_Neo_LowerECAM_Engine;
         }
     }
     A320_Neo_LowerECAM_Engine.EngineInfo = EngineInfo;
+    class EngineIgniter {
+        constructor(_ignValueText) {
```

Comment:
**thought**: We are in the `EngineIgniter` so a `ign` prefix might be duplicate information. The name isn't telling me it's an element.
**suggestion**: Consider renaming this to something like `textElement` (or maybe you have a better name? 😀)

Reply:
Done :)